docs/source/contribs/development_Mac.md

```diff
@@ -0,0 +1,80 @@
+# Tensorcircuit Installation Guide on MacOS
+
+Contributed by [_Mark (Zixuan) Song_](https://marksong.tech)
+
+Apple has updated Tensorflow (for MacOS) so that installation on M-series (until M2) and Intel-series Mac can follow the exact same procedure.
+
+## Starting From Scratch
+
+For completely new Macos or Macos without Xcode and Homebrew installed.
+
+### Install Xcode Command Line Tools
+
+<font color=gray><em>Need graphical access to the machine.</em></font>
+
+Run `xcode-select --install` to install if on optimal internet.
+
+Or Download it from [Apple](https://developer.apple.com/download/more/) Command Line Tools installation image then install it if the internet connection is weak.
+
+## Install Miniconda
+
+Due to the limitation of MacOS and packages, the latest version of Python does not always function as desired, thus miniconda installation is advised to solve the issues.
+
+```bash
+curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh
+bash ~/miniconda.sh -b -p $HOME/miniconda
+source ~/miniconda/bin/activate
+conda install -c apple tensorflow-deps
+```
+
+## Install TC Backends
+
+There are four backends to choose from, Numpy, Tensorflow, Jax, and Torch.
+
+### Install Jax, Pytorch, Qiskit, Cirq (Optional)
+
+```bash
+pip install [Package Name]
+```
+
+### Install Tensorflow (Optional)
+
+#### Installation
+
+For Tensorflow version 2.13 or later:
+```bash
+pip install tensorflow
+pip install tensorflow-metal
+```
+
+For Tensorflow version 2.12 or earlier:
+```bash
+pip install tensorflow-macos
```

Comment:
is this work for intel mac?

Reply:
Yes

Reply:
are you indicating that there is a package tensorflow-macos also on intel mac？ I do think install tensorflow is the best solution for intel mac for these early versions

Reply:
It should be updated and remove install guide for tensorflow version 2.12 or earlier. For tf of those versions, tf-macos and vanilla tf are both not functioning as desired on tested platforms.

Reply:
updated and please refer to the latest updated version as this issues have been resolved in latest update.